theorems/T000298.md

```diff
@@ -0,0 +1,14 @@
+---
+uid: T000298
+if:
+  and:
+    - P000138: true
+    - P000005: true
+then:
+  P000078: true
+refs:
+  - mo: 4231158
+    name: Is there an infinite topological space with only countably many continuous maps to itself?
+---
+
+See [an anonymous comment](https://mathoverflow.net/questions/418619/is-there-an-infinite-topological-space-with-only-countably-many-continuous-funct#comment1075675_418619) in {{mathse:4231158}}.
```

Comment:
I think it would be useful to mention that this uses the intermediate step that metrizable spaces with P138 must be finite, which is shown in https://math.stackexchange.com/questions/4231158.  And add that as an additional reference. (otherwise may be missed by a casual reader)